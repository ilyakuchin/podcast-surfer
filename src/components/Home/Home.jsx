import React from 'react';
import exploreImage from './undraw_imagination_ok71.svg';

export default function Home() {
  return (
    <div>
      <div>
        <div>
          <img src={exploreImage} />
        </div>
        <div>
          <h2>Discover audio stories, news and shows</h2>
        </div>
      </div>
    </div>
  );
}
