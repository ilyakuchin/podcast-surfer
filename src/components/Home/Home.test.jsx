import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from './Home';

configure({ adapter: new Adapter() });

describe('Test Home component', () => {
  test('should render correctly', () => {
    expect(toJson(shallow(<Home />))).toMatchSnapshot();
  });
});
