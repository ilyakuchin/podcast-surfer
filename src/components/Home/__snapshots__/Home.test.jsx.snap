// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home component should render correctly 1`] = `
<div>
  <div>
    <div>
      <img
        src="undraw_imagination_ok71.svg"
      />
    </div>
    <div>
      <h2>
        Discover audio stories, news and shows
      </h2>
    </div>
  </div>
</div>
`;
